Tests/SwiftLintFrameworkTests/ReporterTests.swift

```diff
@@ -92,8 +92,12 @@ class ReporterTests: SwiftLintTestCase {
     }
 
     func testCodeClimateReporter() {
-        let expectedOutput = stringFromFile("CannedCodeClimateReporterOutput.json")
-        let result = CodeClimateReporter.generateReport(generateViolations())
+        let expectedOutput = stringFromFile("CannedCodeClimateReporterOutput.json").filter {
+            !$0.isWhitespace
+        }
+        let result = CodeClimateReporter.generateReport(generateViolations()).filter {
+            !$0.isWhitespace
+        }
```

Comment:
Why is the filtering suddenly necessary? It's just a few strings that have changed their case and need an update in the reference file.

Reply:
I am not sure why i was struggling so hard to get all the whitespace to match, ill give it another shot

Reply:
Got it! resolved, removed these whitespace filters